app/src/test/java/fr/free/nrw/commons/TestCommonsApplication.java

```diff
@@ -15,7 +15,6 @@
 import fr.free.nrw.commons.data.DBOpenHelper;
 import fr.free.nrw.commons.di.CommonsApplicationComponent;
 import fr.free.nrw.commons.di.CommonsApplicationModule;
-import fr.free.nrw.commons.di.DaggerCommonsApplicationComponent;
 import fr.free.nrw.commons.location.LocationServiceManager;
```

Comment:
It seems like removing this import results Travis CI fail. Why did you decided to remove this line?

Reply:
sorry, my bad; I did not intentionally do it but when committing/pushing, Android Studio organized the imports, and since it's a compile-time generated class and it wasn't there, he removed it. I will put it back immediately

Reply:
and yet it still fails